backend/btrixcloud/models.py

```diff
@@ -739,7 +746,6 @@ class CrawlFileOut(BaseModel):
 
     crawlId: Optional[str] = None
     numReplicas: int = 0
-    expireAt: Optional[str] = None
```

Comment:
I wonder if it might be good to keep this in the response. I know we've had a few folks using the presigned URLs in webhook notifications to download content, they might appreciate knowing how long the presigned URL is valid for.

This is slightly complicated now by the TTL being set at 75% of expiry and the inexact timing of when the cleanup happens exactly once the TTL is reached, but maybe returning the timestamp for when the TTL is set to be hit would be helpful?

Reply:
Ah, I forgot that the expireAt is also included in webhooks -- otherwise, its not really needed, the immediate response should have plenty of time left. It was actually set to 75% of actual expiry time already if you look at `expire_at_duration_seconds`, so it could be added back in I suppose..